Automattic/pocket-casts-android
# [Paywall Experiment] Add Reviews section to Upgrade Layout Reviews

## Description
- This adds the reviews to Upgrade Layout Reviews
- See figma: SEQe6JBDCeegcpa9Bfg5Ww-fi-4_1455

> [!NOTE]  
> The reviews are mocked and added to string.xml in order to get translated 

> [!NOTE]  
> On Android, We are going to display `See all reviews in the Play Store` instead of `See all reviews in the App Store`


Fixes #2710

## Testing Instructions
1. Apply this [paywall.patch](https://github.com/user-attachments/files/16612993/paywall.patch) and run in debug
2. Have `PAYWALL_AB_EXPERIMENT` feature flag enabled
3. Open paywall
4. ✅ Ensure the layout works as expected in Figma

## Screenshots or Screencast 

[Screen_recording_20240828_135935.webm](https://github.com/user-attachments/assets/42af6b1e-be4e-448a-a506-fbfc013179bc)

| Reviews part 1 | Reviews part 2 | Reviews part 3 |
|--------|--------|--------|
| ![Screenshot_20240828_135733](https://github.com/user-attachments/assets/70714981-98db-499d-bc05-b58f0bb117ea) | ![Screenshot_20240828_135743](https://github.com/user-attachments/assets/c4ea7ec1-8745-4758-90b2-cdf4cb827cbd) | ![Screenshot_20240828_135751](https://github.com/user-attachments/assets/5f0ac9ad-ddfc-4a70-a86a-88e0cfe44336) | 


## Checklist
 
#### I have tested any UI changes...
